.gitignore

```diff
@@ -1,6 +1,7 @@
 node_modules
 lcov.info
 server.json
-.vscode
```

Comment:
Can we please add this back in .vscode. Or just delete the extensions.json, launch.json, settings.json and tasks.json. I guess the whole folder is not needed.

Can we also delete asb folder please. It is not needed.